test/lib/graph/graph.js

```diff
@@ -5,17 +5,20 @@ import sinonGlobal from "sinon";
 import esmock from "esmock";
 const __dirname = path.dirname(fileURLToPath(import.meta.url));
 
+const fixturesPath = path.join(__dirname, "..", "..", "fixtures");
```

Comment:
could be moved one line above, so only one empty line after imports and consts.

Reply:
I like to keep some space between imports and consts. So I moved the __dirname const by one line

Reply:
Ack